docs/source/move_mode.rst

```diff
@@ -81,9 +81,16 @@ afew will only check mails at most 15 days old.
 Rename
 ------
 
-When afew moves mail files, if the the `rename` option in the configuration is
-set afew will rename those files. You should probably do this if you're using
-the `mbsync` program to interact with your IMAP server.
+Set this option if you are using the `mbsync` IMAP syncing tool.
+`mbsync` adds a Unique IDentifier to files' names when it syncs them.
```

Comment:
typo in `IDentifier`

Reply:
I did that on purpose, trying to be clear about what "UID" meant. Not particularly attached to it, though — I was just trying to explain the feature without diving too deep into `mbsync` implementation details.